File: ruby/iniciante/1036.rb
coef = gets.chomp.split(' ')
A = coef[0].to_f
B = coef[1].to_f
C = coef[2].to_f
Delta = (B ** 2) + (-4 * A * C)

if Delta <= 0 || A == 0
	puts "Impossivel calcular"
else
	R1 = (- B + Math.sqrt(Delta)) / (2 * A)
	R2 = (- B - Math.sqrt(Delta)) / (2 * A)
	puts "R1 = "'%.5f' % R1
	puts "R2 = "'%.5f' % R2
end

File: python3/iniciante/1036.py
import math

a, b, c = list(map(float, input().split()))

delta = b**2 - 4*a*c

if delta < 0 or a == 0:
    print("Impossivel calcular")
else:
    r1 = (-b + math.sqrt(delta)) / (2 * a)
    r2 = (-b - math.sqrt(delta)) / (2 * a)

    print("R1 = {:.5f}".format(r1))
    print("R2 = {:.5f}".format(r2))
